components/phpunit_bridge.rst

```diff
@@ -0,0 +1,114 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report legacy tests and
+    usage of deprecated code.
+
+It comes with the following features:
+
+* Forces the application to use the default language locale for output (set the
+  locale to ``C``)
+* Auto-register ``class_exists`` to load Doctrine annotations (when used)
+* It displays the whole list of deprecated features used in the application
+* Display the stack trace of a deprecation on-demand.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>`
+  (``symfony/phpunit-bridge`` on `Packagist`_); as a dev dependency
+* Use the official Git repository (https://github.com/symfony/phpunit-bridge)
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+Once the component installed, it automatically registers a
+`PHPUnit event listener`_ which in turn registers a `PHP error handler`_
+called ``DeprecationErrorHandler``. After running your PHPUnit tests again, you
+will get a report similar to this one:
+
+.. image:: /images/components/phpunit_bridge/report.png
+
+The summary includes:
+
+* **Unsilenced** reports deprecation notices that were triggered without the
+  recommended @-silencing operator
+* **Legacy** deprecation notices denote tests that explicitly test some legacy
+  features
+* **Remaining/Other** deprecation notices are all other (non-legacy) notices,
+  grouped by message, test class and method
+
+Trigger deprecation notices
+---------------------------
+
+Deprecation notices can be triggered by using::
+
+    @trigger_error('Your deprecation message', E_USER_DEPRECATED);
+
+Without the @-silencing operator, users would need to opt-out from deprecation
+notices. Silencing by default swaps this behavior and allows users to opt-in
+when they are ready to cope with them (by adding a custom error handler like the
+one provided by this bridge). When not silenced, deprecation notices will appear
+in the **Unsilenced** section of the deprecation report.
+
+Mark tests as legacy
+--------------------
+
+There are four ways to mark a test as legacy:
+
+* Make its class start with the ``Legacy`` prefix
+* Make its method start with ``testLegacy``
+* Make its data provider start with ``provideLegacy`` or ``getLegacy``
+* Add the ``@group legacy`` annotation to its class or method
```

Comment:
this is the most recommended way, let's put it as the first item (and maybe add `(**recommended**)`)